src/ucioption.cpp

```diff
@@ -64,7 +64,7 @@ void init(OptionsMap& o) {
   o["Clear Hash"]            << Option(on_clear_hash);
   o["Ponder"]                << Option(true);
   o["MultiPV"]               << Option(1, 1, 500);
-  o["Skill Level"]           << Option(20, 0, 20);
+  o["Skill Level"]           << Option(40, 1, 40);
```

Comment:
Why is the max 40 ? max should verify the equation:

    1 + 3 * max / 4 = 1 + 20
    max = 4 * 20 / 3 ~= 27

Have you looked at the massive diminishing return effect at level 15 already ? 27 is enough. After that you create fake levels that have practically the same elo anyway.